src/conversion/model-to-view-converters.js

```diff
@@ -252,9 +254,11 @@ export function removeAttribute( attributeCreator ) {
  *
  * @param {module:engine/view/element~Element|Function} elementCreator View element, or function returning a view element, which will
  * be used for wrapping.
+ * @param {Function} [nameToConsumable] Optional function used to map conversion event name to consumable name. By default it
+ * uses {engine/conversion/model-to-view-converters~eventNameToConsumableType} function.
  * @returns {Function} Set/change attribute converter.
  */
-export function wrapItem( elementCreator ) {
+export function wrapItem( elementCreator, nameToConsumable = eventNameToConsumableType ) {
```

Comment:
My problem is not the fact that we add a callback. My problem is that translating names to consumable was very very simple (see https://github.com/ckeditor/ckeditor5-engine/blob/2ccd95ff89e7d8522891a925a034056959f145b6/src/conversion/model-to-view-converters.js#L544-L554). You did not need to think much when you see `nameToConsumable` in the code. Now if it can be a callback we bring another level of abstraction to already very complex part of the code. We should think twice if we really need to do it.